CONTRIBUTING.md

```diff
@@ -97,17 +97,16 @@ you can enable the docker tests by setting a gradle property of
 ``"enable.docker.tests"`` to true. See the gradle.properties file in the root of the project
 for more details.
 
-2. From the root project directory, initialize repository dependencies
+2. Clone the repository recursively
 
-   `make init-git-submodules`
+    `git clone git@github.com:<username>/opentelemetry-java.git --recursive`
```

Comment:
I think recommending SSH clones is asking for trouble. Usually HTTPS is easier.

Reply:
Also, either explicitly say that a fork should be used, or just use the full URL `https://github.com/open-telemetry/opentelemetry-java.git`